python/cudf/cudf/core/dataframe.py

```diff
@@ -1154,6 +1154,12 @@ def _get_renderable_dataframe(self):
 
         if len(self) <= nrows and len(self._data.names) <= ncols:
             output = self.copy(deep=False)
+        elif self.empty and len(self.index) > 0:
+            # Incase of Empty DataFrame with index, Pandas prints
+            # first 100 index values followed by ...
+            # To obtain ... at the end of index list, adding 2 more
+            # values, hence fetching only first 102 values.
+            output = self.head(102)
```

Comment:
Is this not dependent on an option like `pd.options.display.max_rows` for example?

Reply:
Surprisingly nope, tried playing around with options like max_rows, max_columns. Changing those values(ranging from 0, 2, 4, 100, 1000) made no difference for index representation incase of an empty dataframe

Reply:
I think there is one, `pd.options.display.max_seq_items` changing the code accordingly

Reply:
Changed the logic to rely on `max_seq_items`